WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        public static User User { get; set; }

```

Comment:
I couldn't tell for sure, but looking I the code I don't see reasons not to. Do you think it might have issues?